src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java

```diff
@@ -327,8 +327,16 @@ public int getId()
      */
     class Choice
     {
+        /**
+         * The maximum length the name of a choice can be.
+         */
         public static final int MAX_NAME_LENGTH = 100;
 
+        /**
+         * The maximum length the value of a choice can be.
+         */
+        public static final int MAX_VALUE_LENGTH = 100;
```

Comment:
I wonder if we need to change this to `MAX_STRING_VALUE_LENGTH` to differentiate it from other value types. Kinda ambiguous currently.